src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,10 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+import './UserInfo.scss';
+
+export const UserInfo: React.FC<User> = ({ name, email }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${email}`}>
+      {name}
+    </a>
+  </>
```

Comment:
_comment_
```suggestion
  <a className="UserInfo" href={`mailto:${email}`}>
    {name}
  </a>
```
there is no actual need in React fragment when you only have one element

Reply:
thanks, missed it